configs/mvpose/README.md

```diff
@@ -114,22 +114,22 @@ The CMU Panoptic dataset contains four sequences that share the same config file
 
 | Config | Actor 0 | Actor 1 | Actor 2 | Average | Download |
```

Comment:
For CMU Panoptic, shall we keep only the mean PCP and add more metrics results we got from our evaluator? As there is no paper reporting the per-actor PCP for comparison, MPJPE is more commonly used for evaluating the Panoptic dataset.